MANIFEST.in

```diff
@@ -1,6 +1,7 @@
 include README.rst
 include LICENSE
 include tox.ini
+include mypy.ini
```

Comment:
Let's add the py.typed files as in https://github.com/mongodb/mongo-python-driver/commit/80314255d7fb10769a083fcbee3d613a7263bd92

As well as the "Typing :: Typed" classifier to setup.py.